src/main/java/io/github/yeffycodegit/poglog/PogLogger.java

```diff
@@ -0,0 +1,83 @@
+package io.github.yeffycodegit.poglog;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class PogLogger {
+    private final boolean logToFile;
+    private String filename;
+    FileWriter fileWriter;
+
+    // Define color constants
```

Comment:
Here, you should take these colour constants and place them into a different class, maybe ColourConstants.java?

Reply:
Okay. Will fix them in a sec.